README.md

```diff
@@ -31,14 +31,16 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
 * `categories`: the categories that it will be shown under. Choose at least one category from the list [here](#categories). Further, choose the categories from the list of languages available [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). When a user views the available templates, those templates that match the same language will feature more prominently.
 
 ### Categories
-* continuous-integration
-* deployment
-* testing
+* automation
+* ci
 * code-quality
 * code-review
+* code-scanning
+* continuous-integration
 * dependency-management
+* deployments
```

Comment:
```suggestion
* deployment
```

Reply:
The repo contains directory `deployments` not `deployment`

Reply:
Yes but the category is `deployment`. I also just got this difference 🤔